dictionaries/dict.json

```diff
@@ -108460,7 +108459,7 @@
 "THAOERT": "theater",
 "THAOERT/EUBG": "theatric",
 "THAOERT/EUBG/-S": "theatrics",
-"THAOERT/K-L": "theoretical",
```

Comment:
Let's add `"THAOERT/K-L": "theoretical",` to `bad-habits.json`

Reply:
Will do.